schematic/store/synapse.py

```diff
@@ -606,7 +661,10 @@ def getProjectManifests(self, projectId: str) -> List[str]:
                         )
 
                     manifest_info = self.getDatasetManifest(datasetId,downloadFile=True)
-                    manifest_name = manifest_info["properties"]["name"]
+                    try:
+                        manifest_name = manifest_info["properties"]["name"]
```

Comment:
Are there ever scenarios where `manifest_info['properties']` doesn't exist?  if not, how about doing:

```
manifest_name = manifest_info["properties"].get("name", "")
```

You can then remove this later

```
manifest_name = ""
```

Reply:
Thanks for the suggestion Tom. If I use: `manifest_name = manifest_info["properties"].get("name", "")`, I don't need the try except?

Reply:
Well it seems like you still want to log the fact that you failed to download the manifest which is why this exists.  What happens when you fail to download the manifest, does `load_df` work?

Reply:
No, I don't think load_df would work since that requires "path" as a key. Maybe I could try something like: 
```
                    manifest_name = manifest_info["properties"].get("name", "")

                    if not manifest_name:
                        logger.error(f'Failed to download manifests from {datasetId}') 
```